packages/connector-aliyun-sms/src/index.ts

```diff
@@ -25,12 +27,27 @@ export default class AliyunSmsConnector implements SmsConnector<AliyunSmsConfig>
     }
   }
 
-  public sendMessage: SmsSendMessageFunction = async (phone, type, { code }, config) => {
-    const smsConfig = config ?? (await this.getConfig(this.metadata.id));
+  public sendMessage: SmsSendMessageFunction = async (phone, type, data) => {
+    const smsConfig = await this.getConfig(this.metadata.id);
 
     this.validateConfig(smsConfig);
 
-    const { accessKeyId, accessKeySecret, signName, templates } = smsConfig;
+    return this.sendMessageBy(phone, type, data, smsConfig);
+  };
+
+  public sendTestMessage: SmsSendTestMessageFunction = async (config, phone, type, data) => {
+    this.validateConfig(config);
```

Comment:
nit: can we put `validateConfig` in the private `sendMessageBy` function?

Reply:
discussed offline, will do later